### PR TITLE
.github/workflows: Add python checks for support scripts

### PR DESCRIPTION
**Pull request by craciunoiuc**
### Prerequisite checklist

 - [x] Read the [contribution guidelines](https://unikraft.org/docs/contributing/) regarding submitting new changes to the project;
 - [ ] Tested your changes against relevant architectures and platforms;
 - [ ] Ran the [`checkpatch.uk`](https://github.com/unikraft/unikraft/blob/staging/support/scripts/checkpatch.uk) on your commit series before opening this PR;
 - [ ] Updated relevant documentation.

### Base target

 - Architecture(s): [e.g. `x86_64` or N/A]
 - Platform(s): [e.g. `kvm`, `xen` or N/A]
 - Application(s): [e.g. `app-python3` or N/A]

### CODE REVIEW

File: .github/workflows/pychecks.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      - uses: actions/checkout@v3
+
+      - name: black
+        uses: psf/black@stable
+        with:
+          src: support/scripts
+
+      - name: ruff
+        uses: chartboost/ruff-action@v1
+        if: success() || failure()
```
Comment (jschlumpp) [inline .github/workflows/pychecks.yaml:30]: What about running this in a different job instead?

Comment (craciunoiuc) [reply .github/workflows/pychecks.yaml:30]: We would need to double-checkout, but the advantage is that they will run in parallel (+ show up separately)

Comment (craciunoiuc) [reply .github/workflows/pychecks.yaml:30]: I'm fine either way

**Review by razvand — APPROVED** [review]
Reviewed-by: Razvan Deaconescu <razvand@unikraft.io>
Approved-by: Razvan Deaconescu <razvand@unikraft.io>

**Pull request merged**